scripts/src/main/resources/scripts/command/lazydocker

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup help"
+  fi
+  exit
+fi
+
+if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+
+# Call lazydocker with specified arguments.
+function doRun() {
+  doSetup silent
+  cd "${LAZYDOCKER_HOME}" || exit 1
+  if doIsWindows
+  then
+    powershell.exe -command 'start powershell {.\lazydocker.exe}'
+  else
+    ./lazydocker 
+  fi
+}
+
+function doSetup() {
+  if [ ! -d "${LAZYDOCKER_HOME}" ] || [ "${1}" != "silent" ]
```

Comment:
you should add
```
doDevonCommand docker setup silent
```
as before what will ensure that docker/rancher is first installed if not already present.

Reply:
The lazydocker cmdlet should work, but this additional command seems to make problems in my case. For Antigoni it is fine again. I suspect that this is the case because of some changes that I made for testing purposes, but I will look deeper into this next week and make another commit if necessary.